telegram/_poll.py

```diff
@@ -126,13 +126,18 @@ class Poll(TelegramObject):
         is_anonymous (:obj:`bool`): :obj:`True`, if the poll is anonymous.
         type (:obj:`str`): Poll type, currently can be :attr:`REGULAR` or :attr:`QUIZ`.
         allows_multiple_answers (:obj:`bool`): :obj:`True`, if the poll allows multiple answers.
-        correct_option_id (:obj:`int`, optional): 0-based identifier of the correct answer option.
-            Available only for polls in the quiz mode, which are closed, or was sent (not
-            forwarded) by the bot or to the private chat with the bot.
+        correct_option_id (:obj:`int`, optional)): A zero based identifier of the correct answer option.
+            Available only for closed polls in the quiz mode, which were sent (not
+            forwarded), by the bot or to a private chat with the bot.
         explanation (:obj:`str`, optional): Text that is shown when a user chooses an incorrect
-            answer or taps on the lamp icon in a quiz-style poll, 0-200 characters.
-        explanation_entities (List[:class:`telegram.MessageEntity`], optional): Special entities
+            answer or taps on the lamp icon in a quiz-style poll, 0-200 characters with at most
+            2 line feeds after entities parsing.
```

Comment:
Please remove the "2 line feeds..." senetence

Reply:
done